src/folio_migration_tools/mapping_file_transformation/manual_fee_fines_mapper.py

```diff
@@ -0,0 +1,276 @@
+            )
+
+        elif folio_prop_name == "account.feeFineId" and self.feefines_type_map:
+            return self.get_mapped_ref_data_value(
+                self.feefines_type_map, legacy_object, index_or_id, folio_prop_name, False
+            )
+
+        elif folio_prop_name == "account.userId" or folio_prop_name == "feefineaction.userId":
+            return self.get_matching_record_from_folio(
+                index_or_id,
+                self.user_cache,
+                "/users",
+                "barcode",
+                super().get_prop(
+                    legacy_object, folio_prop_name, index_or_id, schema_default_value
+                ),
+                "users",
+            )["id"]
+
+        elif folio_prop_name == "feefineaction.id":
+            return str(uuid.uuid4())
+
+        elif folio_prop_name == "feefineaction.createdAt" and self.service_point_map:
+            return self.get_mapped_ref_data_value(
+                self.service_point_map, legacy_object, index_or_id, folio_prop_name, False
+            )
+
+        elif folio_prop_name == "feefineaction.dateAction":
+            return self.parse_date(
+                folio_prop_name,
+                index_or_id,
+                super().get_prop(
+                    legacy_object, folio_prop_name, index_or_id, schema_default_value
+                ),
+                legacy_object,
+            )
+
+        elif mapped_value := super().get_prop(
+            legacy_object, folio_prop_name, index_or_id, schema_default_value
+        ):
+            return mapped_value
+
+        else:
+            return ""
+
+    def get_composite_feefine_schema(self) -> Dict[str, Any]:
+        return {
+            "properties": {
+                "account": FolioClient.get_latest_from_github(
+                    "folio-org", "mod-feesfines", "/ramls/accountdata.json"
+                ),
+                "feefineaction": FolioClient.get_latest_from_github(
+                    "folio-org", "mod-feesfines", "/ramls/feefineactiondata.json"
+                ),
+            }
+        }
+
+    def get_matching_record_from_folio(
+        self,
+        index_or_id,
+        cache: dict,
+        path: str,
+        match_property: str,
+        match_value: str,
+        result_type: str,
+    ):
+        if match_value not in cache:
+            query = f'?query=({match_property}=="{match_value}")'
+            if matching_record := next(
+                self.folio_client.folio_get_all(path, result_type, query), None
+            ):
+                cache[match_value] = matching_record
+                return matching_record
+            else:
+                raise TransformationFieldMappingError(
+                    index_or_id, f"No matching {result_type} for {match_property}", match_value
+                )
+        else:
+            return cache[match_value]
+
+    def parse_date(self, folio_prop_name: str, index_or_id, mapped_value, legacy_object):
```

Comment:
@branchedelac Do we want to do any local timezone stuff here like we did with loans, or can we just assume UTC if no timezone indications is present in the source data?

Reply:
https://github.com/FOLIO-FSE/folio_migration_tools/issues/619

Reply:
Were you thinking you'd include that in this PR, or create a new feature branch?

Reply:
Ok. I'll merge and we can move from there. I'm going to rebase, so you'll want to either wait to create the branch or rebase to main after this is merged.